start-prod.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+npx npx prisma db push
```

Comment:
I think prisma migrate would be more appropriate.
Also, lose the double npx

Reply:
https://www.prisma.io/docs/orm/prisma-client/deployment/deploy-database-changes-with-prisma-migrate
It says that for mongoDB instead of migrate we should use db push

Reply:
Then just fix the double npx